pkg/util/config.go

```diff
@@ -37,3 +40,62 @@ func DefaultResourceThresholdStrategy() *slov1alpha1.ResourceThresholdStrategy {
 		MemoryEvictThresholdPercent: pointer.Int64Ptr(70),
 	}
 }
+
+func DefaultResourceQoSStrategy() *slov1alpha1.ResourceQoSStrategy {
+	return &slov1alpha1.ResourceQoSStrategy{
+		Enable: pointer.BoolPtr(false), // disable resource QoS by default
+		LSRClass: &slov1alpha1.ResourceQoS{
+			ResctrlQoS: &slov1alpha1.ResctrlQoSCfg{
+				Enable:     pointer.BoolPtr(false),
+				ResctrlQoS: *DefaultResctrlQoS(extension.QoSLSR),
+			},
+		},
+		LSClass: &slov1alpha1.ResourceQoS{
+			ResctrlQoS: &slov1alpha1.ResctrlQoSCfg{
+				Enable:     pointer.BoolPtr(false),
+				ResctrlQoS: *DefaultResctrlQoS(extension.QoSLS),
+			},
+		},
+		BEClass: &slov1alpha1.ResourceQoS{
+			ResctrlQoS: &slov1alpha1.ResctrlQoSCfg{
+				Enable:     pointer.BoolPtr(false),
+				ResctrlQoS: *DefaultResctrlQoS(extension.QoSBE),
+			},
+		},
+	}
+}
+
+func NoneResctrlQoS() *slov1alpha1.ResctrlQoS {
+	return &slov1alpha1.ResctrlQoS{
+		CATRangeStartPercent: pointer.Int64Ptr(0),
+		CATRangeEndPercent:   pointer.Int64Ptr(100),
+		MBAPercent:           pointer.Int64Ptr(100),
+	}
+}
+
+func DefaultResctrlQoS(qos extension.QoSClass) *slov1alpha1.ResctrlQoS {
```

Comment:
The default value is the most important best practice principle, and we need to write a paragraph of design ideas here to guide users to personalize configuration. or leave a TODO here.

Reply:
Done